translator/translate/otel/receiver/otlp/translator.go

```diff
@@ -73,6 +74,97 @@ func (t *translator) ID() component.ID {
 	return component.NewIDWithName(t.factory.Type(), t.Name())
 }
 
+var (
+	registryMu sync.RWMutex
```

Comment:
Do we need synchronization? I thought all translation was done on a single thread.